Traceback (most recent call last):
  File "/home/nader/projects/lifted-white-plan-domain-repair/instance_solver.py", line 109, in <module>
    solve_instance(
  File "/home/nader/projects/lifted-white-plan-domain-repair/instance_solver.py", line 35, in solve_instance
    gr = ground_repair(
         ^^^^^^^^^^^^^^
  File "/home/nader/projects/lifted-white-plan-domain-repair/vanilla_runs/run_songtuans_vanilla.py", line 22, in ground_repair
    raise "Problem is unsolvable"
TypeError: exceptions must derive from BaseException
Traceback (most recent call last):
  File "/home/nader/projects/lifted-white-plan-domain-repair/instance_solver.py", line 109, in <module>
    solve_instance(
  File "/home/nader/projects/lifted-white-plan-domain-repair/instance_solver.py", line 35, in solve_instance
    gr = ground_repair(
         ^^^^^^^^^^^^^^
  File "/home/nader/projects/lifted-white-plan-domain-repair/vanilla_runs/run_songtuans_vanilla.py", line 22, in ground_repair
    raise "Problem is unsolvable"
TypeError: exceptions must derive from BaseException
Traceback (most recent call last):
  File "/home/nader/projects/lifted-white-plan-domain-repair/instance_solver.py", line 109, in <module>
    solve_instance(
  File "/home/nader/projects/lifted-white-plan-domain-repair/instance_solver.py", line 35, in solve_instance
    gr = ground_repair(
         ^^^^^^^^^^^^^^
  File "/home/nader/projects/lifted-white-plan-domain-repair/vanilla_runs/run_songtuans_vanilla.py", line 22, in ground_repair
    raise "Problem is unsolvable"
TypeError: exceptions must derive from BaseException
